libraries/testbot/dialogs/cancelAndHelpDialog.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ComponentDialog, DialogTurnStatus } = require('botbuilder-dialogs');
+
+/**
+ * This base class watches for common phrases like "help" and "cancel" and takes action on them
+ * BEFORE they reach the normal bot logic.
+ */
+class CancelAndHelpDialog extends ComponentDialog {
```

Comment:
Add unit tests for this base class.

Sample c# tests can be found here:
https://github.com/microsoft/BotBuilder-Samples/blob/8dcaafc51a73f6663c7f53d07f3c9eaec0b68d5d/samples/csharp_dotnetcore/13.core-bot.tests/Dialogs/CancelAndHelpDialogTests.cs

Reply:
done

Reply:
Reopened, we need an individual test for the dialog. I am working on it.

Reply:
done